skyportal/handlers/api/followup_request.py

```diff
@@ -76,25 +77,26 @@ def post_assignment(data, user_id, session):
 
     run_id = assignment.run_id
     data['priority'] = assignment.priority.name
-    run = ObservingRun.get_if_accessible_by(run_id, user, raise_if_none=False)
+    run = session.scalars(
+        ObservingRun.select(user).where(ObservingRun.id == run_id)
```

Comment:
And the `user` comes from an old style query...

Reply:
Maybe safer to use `session.user_or_token` unless you are specifically allowing people to post assignments with other user's permissions.

Reply:
agreed!